docs/advanced-usage/04-dynamic-logging-level-threshold.md

```diff
@@ -30,7 +30,14 @@ Using this feature allows changing the logging level threshold dynamically witho
 ### 1 Creating a key-pair
 
 To sign and verify JWTs a PEM encoded private key and a matching public key is required.
-You can create a key-pair using the following command:
+
+You can create a private key using the following command:
+
+```sh
+openssl genrsa -des3 -out private.pem 2048
```

Comment:
Please remove the -des3 parameter.
An encrypted private key is typically impractical for the intended use case. However, I agree that we should support it general for debugging/poc purposes.